src/ILCompiler.Compiler/src/Compiler/CompilerTypeSystemContext.cs

```diff
@@ -231,7 +234,7 @@ private unsafe static PEReader OpenPEFile(string filePath, out MemoryMappedViewA
                 if (accessor != null)
                     accessor.Dispose();
                 if (mappedFile != null)
-                    mappedFile.Dispose();
+                    mappedFile.Dispose(); // Cleans up the fileStream.
```

Comment:
I see - I did not get it from the comment. It may useful to change it to "Closes the fileStream as well".

BTW: It does not matter a whole lot here ... but it would be more robust to explicitly dispose the fileStream and not depend on mappedFile to do it - if MemoryMappedFile.CreateFromFile throws exception, the fileStream won't get closed (the finalizer will take care of it later though).